layouts/page/guideline.html

```diff
@@ -0,0 +1,204 @@
+{{- define "main" -}}
+    {{ partial "nav.html" (dict "nav_white" 1) }}
+    {{ $guideline := .Site.Data.guideline }}
+
+    <section style="padding-top: 6.5rem" class="mb-5">
+        <div class="container">
+            <div class="d-flex row justify-content-between align-items-center">
+                <!-- Left Image -->
+                <div class="col-md-2 text-center d-none d-md-block ms-1 me-0 mb-0 px-0 py-0" style="width: 15%">
+                    <img style="max-width: 100%;" src='{{ absURL "guideline/side-img-1.png" }}'/>
+                </div>
+                <!-- Center Content -->
+                <div class="col-md-8">
+                    <h2 class="text-center text-primary fw-lighter">
+                        Know each <span class="fw-bold">department</span>.
+                    </h2>
+                    <p class="text-center">
+                        Find out more about the mission and general tasks of
+                        each department.
+                    </p>
+                </div>
+                <!-- Right Image -->
+                <div class="col-md-2 text-center d-none d-md-block me-1 px-3" style="width: 15%">
+                    <img style="max-width: 100%;" src='{{ absURL "guideline/side-img-2.png" }}' />
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!--list of departments-->
+    <section class="bg-body-secondary mb-5">
+        <div class="container mt-2 py-5">
+            <!-- Executive Director row -->
+            <div class="row justify-content-center">
+                {{ $depFirstRow := slice "Executive Director" "Secretarial Director" }}
```

Comment:
@stingynoodles any reasons for this?

![image](https://github.com/user-attachments/assets/dd0f66e9-ef88-493e-bba2-99826482498e)

Reply:
hmm when I run it locally, everything seems fine, I had aidil checked and its working fine as well, could it be another problem instead of the code?

<img width="954" alt="Screenshot 2024-07-22 222440" src="https://github.com/user-attachments/assets/e09506c2-3d88-441a-9c30-04867e296d83">

Reply:
You're right the "director" should be "department", sorry for not noticing, @aidil-sekandar is this a typo haha

Reply:
ohh yeah, its a mistake. it should be department

Reply:
@stingynoodles 

If so, u can safely delete line 33-46, 50, 58

Reply:
@aidil-sekandar wrote line 33-46 so that the top row only has 2 departments: "Executive Director" and "Secretarial Department"

Reply:
sry, forget bout what i hv said

Reply:
alrr no prob